### PR TITLE
Update api spec - Closes #419

### LINKED ISSUE #419: Update API spec
**Describe the solution you'd like**
Update the SDK API specification to reflect the API changes introduced in Lisk SDK 3.0

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #419, #415

### CODE REVIEW

File: modules/ROOT/pages/reference/api.adoc [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
 == Access Restrictions
 
-The API endpoints are by default restricted to a whitelist of IPs that can be found in `config.json` in the section https://github.com/LiskHQ/lisk/blob/1.0.0/config.json#L35[`api.access.whitelist`].
+The API endpoints are by default restricted to a whitelist of IPs that can be found in `config.json` in the section `api.access.whitelist`.
 If you want your API to be accessable by the public, you can do this by changing `api.access.public` to `true`.
 This will allow anyone to make requests to your Lisk Core node.
 However some endpoints stay private, that means only a list of whitelisted IPs can successfully make API calls to that particular endpoint;
 This includes all forging related API calls.
-By default, only the nodes' local IP is included in the whitelist, you can change the setting in `config.json` in the section https://github.com/LiskHQ/lisk/blob/1.0.0/config.json#L114[`forging.access.whitelist`].
+By default, only the nodes' local IP is included in the whitelist, you can change the setting in `config.json` in the section `forging.access.whitelist`.
```
Comment (chris529) [inline modules/ROOT/pages/reference/api.adoc]:
```suggestion
By default, only the nodes' local IP is included in the whitelist, it is possible to change the setting in `config.json` in the section `forging.access.whitelist`.
```

**Review by chris529 — CHANGES_REQUESTED** [review]
Only one minor issue :-)

File: modules/ROOT/pages/reference/api.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/reference/api.adoc]: I will apply this now, but keep in mind the swagger spec is auto generated from the `swagger.yml` file of the Lisk SDK.

So to fix this for good, it needs to be fixed there.

**Review by chris529 — APPROVED** [review]
Looks good Mona :-)

**Pull request merged**